src/main/java/bio/terra/service/snapshot/flight/create/SnapshotCreateFlight.java

```diff
@@ -292,6 +295,16 @@ public SnapshotCreateFlight(FlightMap inputParameters, Object applicationContext
     // The IAM code contains retries, so we don't make a retry rule here.
     addStep(new SnapshotAuthzIamStep(iamClient, snapshotService, snapshotReq, userReq, snapshotId));
 
+    // add data access control groups to the snapshot
+    List<String> dataAccessControlGroups = snapshotReq.getDataAccessControlGroups();
+    if (Objects.nonNull(dataAccessControlGroups) && !dataAccessControlGroups.isEmpty()) {
+      addStep(
+          new CreateSnapshotGroupConstraintPolicyStep(
+              policyService, snapshotId, dataAccessControlGroups));
```

Comment:
I'm curious about dataAccessControlGroups, I see it's a `List<String>`, should this instead be a `List<Enum>`, I'm just curious about what the dataAccessControlGroups look like?

Reply:
That's a great question! It's not an enum, it's going to be a list of emails where each email represents a group of users

Reply:
Ok, is there any way of making it an email type?

Reply:
The downstream Sam APIs don't specify a type other than `String` for this object. We could add a regex check in our API layer but we'd have to check with the Sam team to get the correct pattern, as I don't think Sam's openapi spec has any regex patterns.